Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json

```diff
@@ -3,41 +3,74 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "Logic_App_Name": {
-            "defaultValue": "IDPSRules-Test1",
+            "defaultValue": "IDPSRulesNotification",
             "type": "String"
         },
-        "connections_office365_1_externalid": {
-            "defaultValue": "/subscriptions/****/resourceGroups/****/providers/Microsoft.Web/connections/office365-1",
-            "type": "String"
+"Sender_Address": {
+            "defaultValue": "",
+            "type": "String",
+  "metadata": {
+                "displayName": "Enter the sender's email address to form the connection to O365 Outlook",
```

Comment:
Display Name: "Enter sender's email address"

Description:  "Enter the sender's email address to form the connection to O365 Outlook"

Reply:
Looks good